lib/Raven/SerializerInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+
+namespace Raven;
+
+interface SerializerInterface
+{
+    /**
+     * Serialize an object (recursively) into something safe for data
+     * sanitization and encoding.
+     *
+     * @param mixed $value
+     * @param int   $max_depth
+     * @param int   $_depth
+     * @return string|bool|double|int|null|object|array
+     */
+    public function serialize($value, $max_depth = 3, $_depth = 0);
```

Comment:
How should I change signature of existing method without breaking back compatibility? @dcramer will grumble